### PR TITLE
Neat 2.0 clean slate

### PR DESCRIPTION
**Pull request by tysongach**
:bomb:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
-.sass-cache
 *gem
 *swp
+.DS_store
+.sass-cache
 Gemfile.lock
+node_modules
 public
 tmp
-contrib/*.css
```
Comment (wardpenney) [inline .gitignore]: After our convo, are we brining this one back?

File: Gulpfile.js [hunk 2]
```diff
@@ -1,20 +1,17 @@
 var gulp = require("gulp"),
   autoprefix = require("gulp-autoprefixer"),
   sass = require("gulp-sass"),
-  connect = require("gulp-connect"),
-  bourbon = require("node-bourbon").includePaths;
+  connect = require("gulp-connect");
```
Comment (wardpenney) [inline Gulpfile.js]: What do you mean hound?

File: README.md [hunk 3]
```diff
@@ -4,9 +4,9 @@
 [![Gitter](http://img.shields.io/badge/gitter-neat-ae3dd2.svg?style=flat)](https://gitter.im/thoughtbot/neat)
 [![Stack Overflow](http://img.shields.io/badge/stack%20overflow-neat-ae3dd2.svg?style=flat)](http://stackoverflow.com/questions/tagged/neat)
 
-## A lightweight, semantic grid framework built with Bourbon
+## A lightweight grid framework
```
Comment (wardpenney) [inline README.md]: niiice

File: .gitignore (hunk 1 above)
Comment (tysongach) [inline .gitignore]: Ah, yes! Great catch.

File: README.md (hunk 3 above)
Comment (tysongach) [inline README.md]: For now…

File: README.md [hunk 4]
```diff
@@ -253,20 +109,29 @@ Unless you [open a pull request](https://github.com/thoughtbot/neat/compare/), t
 - [Bitters](https://github.com/thoughtbot/bitters): Scaffold styles, variables and structure for Bourbon projects
 - [Refills](https://github.com/thoughtbot/refills): Prepackaged patterns and components built with Bourbon, Neat and Bitters
 
+Also check out [Proteus](https://github.com/thoughtbot/proteus), a collection of useful starter kits to help you prototype faster. Each kit comes with Bourbon, Neat and Bitters out-of-the-box.
```
Comment (wardpenney) [inline README.md]: Nice

File: neat.gemspec [hunk 5]
```diff
@@ -1,37 +1,30 @@
...
+  s.add_development_dependency "rspec"
+  s.add_development_dependency "scss_lint", "~> 0.44"
+  s.add_runtime_dependency "sass", "~> 3.4"
+  s.authors = [
+    "Joel Oliveira",
+    "Kyle Fiedler",
+    "Reda Lemeden",
+    "Tyson Gach",
+    "Ward Penney",
+    "Will McMahan"
```
Comment (wardpenney) [inline neat.gemspec]: @joshuaogle too?

File: package.json [hunk 6]
```diff
@@ -24,18 +23,12 @@
...
-  "main": "app/assets/stylesheets/_neat.scss",
+  "main": "index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/thoughtbot/neat.git"
   },
   "scripts": {
     "test": "echo \"No test specified\""
-  },
-  "dependencies": {
```
Comment (wardpenney) [inline package.json]: Nice, we can remove these? Does this remove auto-prefixer from neat?

**Comment by wardpenney** [comment]
A few comments, but I think it's a good move. We can always reference what we need while building the new methods.

**Comment by wardpenney** [comment]
:shipit:

**Comment by tysongach** [comment]
Merged into `neat-2.0.0`

**Pull request closed without merging**